doc/Makefile

```diff
@@ -41,7 +41,9 @@ ifeq ($(wildcard ../info),../info)
 endif
 
 zip:
+ifneq "$(wildcard ../html)" ""
 	@cd .. && zip cc65 html/*.*
+endif
```

Comment:
From your pr comment I understand that the above is rather a fix and not actually related to the change in directory layout - correct?